charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/node-feature-discovery-configmap.yaml

```diff
@@ -967,6 +967,11 @@ data:
           dnsPolicy: ClusterFirstWithHostNet
           securityContext: {}
           serviceAccountName: node-feature-discovery-gc
+          tolerations:
```

Comment:
this is only going to happen when its deployed using cluster resource sets. we would also need to add this bit in the chart values with the helm addon provider.

Reply:
copy these values to this file 

`charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/helm-addon-installation.yaml`

Reply:
Got it. Thanks!

Reply:
Done in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/722/commits/ad60625c904da207da7900862ac3c613bd7493a6